File: server/Hvadskaljegstemme/Models/PartyVote.cs
using Hvadskaljegstemme.Models.Enums;

namespace Hvadskaljegstemme.Models;

public class PartyVote
{
    public int Id { get; set; }

    public Vote Vote { get; set; }

    public required Party Party { get; set; }

    public required Bill Bill { get; set; }
}

File: client/src/lib/types/party-vote.ts
import type { Party } from "./party";
import type { Vote } from "./vote";

export type PartyVoteResponseDTO = {
    partyId: number;
    vote: Vote;
}

export type PartyVote = {
    vote: Vote;
    partyId: number;
    party?: Party;
    billId: number
}